slack_consts.py

```diff
@@ -146,3 +147,12 @@
 
 SLACK_STATE_FILE_CORRUPT_ERR = ("Error occurred while loading the state file due to it's unexpected format. Resetting "
                                 "the state file with the default format. Please test the connectivity.")
+SLACK_SOCKET_TOKEN_ERROR = "Invalid Socket Token please update the configuration file and rerun test connectivity"
```

Comment:
Use ERR instead of ERROR in the variable name for consistency

Reply:
Updated the const variable.

Reply:
Done